From 8a3d86a78f74ec81e182952fd5eced904f39edea Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sun, 21 Jan 2018 00:43:43 +0800
Subject: [PATCH] sunxi: allow to enable MMC driver when using PF UART0

As the pinmux setup of MMC0 at PF is masked, the sunxi MMC driver is now
harmless for PF uart setup, and it may be used for the extra MMC port.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 drivers/mmc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/Kconfig b/drivers/mmc/Kconfig
index 940508364a..ee81b98158 100644
--- a/drivers/mmc/Kconfig
+++ b/drivers/mmc/Kconfig
@@ -365,7 +365,7 @@ config MMC_SDHCI_ZYNQ
 
 config MMC_SUNXI
 	bool "Allwinner sunxi SD/MMC Host Controller support"
-	depends on ARCH_SUNXI && !UART0_PORT_F
+	depends on ARCH_SUNXI
 	default y
 	help
 	  This selects support for the SD/MMC Host Controller on
-- 
2.11.0

